src/components/Button/Button.js

```diff
@@ -2,34 +2,34 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'reakit';
 import styled, { css } from 'styled-components';
-import { tint } from 'polished';
 import tokens from '../../tokens';
 
-export const BaseButton = styled(Button)`
+const BaseButton = styled(
+	React.forwardRef(({ theme, small, ...props }, ref) => <Button ref={ref} {...props} />),
```

Comment:
This is to remove theme and small from dom element attributes.
([Doc](https://styled-components.com/docs/basics#passed-props))